src/main.cpp

```diff
@@ -1,13 +1,19 @@
 #include "core32.h"
 #include "core64.h"
 #include "RNifti_shim.h"
+#ifdef USING_R
+#include "print.h"
+#else
+#include <stdio.h>
+#include <stdlib.h>
+#endif
```

Comment:
Again, the changes to this file don't do anything much, so it would be better to revert them.

Reply:
Agreed - I was trying to use `niimath_print` in `main.cpp` because I was getting segfaults and trying to diagnose, but these can be reverted